truss/build.py

```diff
@@ -280,6 +295,11 @@ def mk_truss(
     )
 
 
+def mk_truss(*args, **kwargs):
```

Comment:
can you please add a `test_backwards` file that tests that mk_truss always calls create with the exact same args? and likewise for load? It would be bad to break this api, even if it's deprecated without having a test fail